.github/workflows/connector-sanity-tests.yml

```diff
@@ -5,10 +5,12 @@ on:
     branches:
       - main
 
-  pull_request:
-
   schedule:
-    - cron: '5 0 * * *'
+    - cron: "5 0 * * *"
```

Comment:
Do we wanna add a comment here explaining the schedule?

Reply:
Since this is a simple one, we can get the explanation from websites like https://crontab.guru/. I'll leave it up to @jarnura as to what we should do here, I'm fine with anything.

Reply:
We will add comments in any upcoming pull request